content/docs/command-reference/metrics/add.md

```diff
@@ -106,7 +104,7 @@ outs:
     path: metrics.json
     cache: true
     metric:
-      type: raw
+      type: json
```

Comment:
This change is correct right @dmpetrov? See https://dvc.org/doc/command-reference/metrics/add#examples

I also removed all mentions of `raw` type metrics in this PR.